src/configs/algo_config.py

```diff
@@ -18,13 +18,13 @@
 
 traditional_fl = {
     "algo": "fedavg",
-    "exp_id": 10,
-    "exp_type": "iid_clients_federated",
+    "exp_id": "no_mixed_precision_more_users",
```

Comment:
can you please make a separate config instead of changing our traditional fl

Reply:
done